### PR TITLE
Add feature bind FQDN and shortname custom ip address.

### PR DESCRIPTION
**Pull request by MemberIT**
For example:
```
---
hostname_ip_address: '{{ ansible_default_ipv4.address }}'
```
Result:
```
# cat /etc/hosts
# Ansible managed

127.0.0.1 localhost.localdomain localhost
default_route_ip fqdn shortname

# The following lines are desirable for IPv6 capable hosts
::1 ip6-localhost ip6-loopback
fe00::0 ip6-localnet
ff00::0 ip6-mcastprefix
ff02::1 ip6-allnodes
ff02::2 ip6-allrouters
ff02::3 ip6-allhosts
```

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -18,11 +18,12 @@ If you just name it with the hostname in the inventory, it will similarly work (
 
 * `hostname_hostname`: [default: `"{{ inventory_hostname }}"`]: FQDN
 * `hostname_hostname_short`: [default: `"{{ inventory_hostname_short }}"`]: Hostname
+* `hostname_use_full`: [default: `false`]: If set to `true` `hostname` equal `FQDN`
+* `hostname_ip_address`: [default: `127.0.1.1`]: `FQDN` and `shortname` resolve to `hostname_ip_address`
```
Comment (tersmitten) [inline README.md]: Can you rename it to `hostname_hostname_ip_address`?

**Comment by tersmitten** [comment]
Thanks for your contribution! I would love to merge this. Can you fix the (minor) remark?

File: README.md (hunk 1 above)
Comment (tersmitten) [reply README.md]: Yes

Comment (MemberIT) [reply README.md]: Done

**Pull request merged**